test/test-bcf-sr.c

```diff
@@ -179,6 +179,7 @@ int main(int argc, char *argv[])
                 break;
             case 'h':
                 usage(EXIT_SUCCESS);
+                // fall-through
```

Comment:
This isn't really a fall-through situation. `usage()` (and `error()`) wants `HTS_NORETURN`.